discovery-infra/start_discovery.py

```diff
@@ -377,12 +377,9 @@ def execute_day1_flow(internal_cluster_name):
         if args.cluster_id:
             cluster = client.cluster_get(cluster_id=args.cluster_id)
         else:
-            random_postfix = str(uuid.uuid4())[:8]
-            ui_cluster_name = internal_cluster_name + f'-{random_postfix}'
-            log.info('Cluster name on UI: %s', ui_cluster_name)
-
             cluster = client.create_cluster(
-                ui_cluster_name, ssh_public_key=args.ssh_key, **_cluster_create_params()
+                cluster_name,
```

Comment:
it means that now all the cluster will be created with the same name. Do we have anywhere a code that uses cluster name to get/find/delete a cluster?